dbt_server/state.py

```diff
@@ -232,11 +277,22 @@ def compile_query(self, query):
 
     @tracer.wrap
     def execute_query(self, query):
+        """Returns a result dict of executing `query` according to current
+        state."""
         return dbt_service.execute_sql(self.manifest, self.root_path, query)
 
     @tracer.wrap
-    def execute_async_command(self, task_id, command, db, callback_url) -> None:
-        return dbt_service.execute_async_command(
+    def execute_async_command(
+        self, task_id: str, command: List[str], db: Any, callback_url: str
+    ) -> None:
+        """Executes core async command. Raises any exception got by core.
+
+        Args:
+            task_id: unique identifier of the task will be executed.
+            command: list of string that will be sent to core.
+            db: local sqlite session, to store task status locally.
+            callback_url: string, will be called back eventually."""
```

Comment:
```suggestion
            callback_url: url string; if provided, calls will be made with new task status on updated."""
```
Not sure what I wrote is any clearer, feel free to tweak!